.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 test/fixtures/*/build
 coverage
+examples/static-site/build
```

Comment:
While normally its true that build directories don't get put in source control, in this particular case I'm actually pretty sure the build directory _is_ supposed to be in the repo.

Since these are meant to be used as examples, it's helpful for users to be able to see the expected result of the build right here on GitHub without having to check out the code and run it themselves.